src/Exception/InvalidNamespaceException.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Exception;
+
+use RuntimeException;
+
+class InvalidNamespaceException extends RuntimeException
```

Comment:
Class doc-block. Even though it's reasonably clear, a one-liner description would be good.